threescale_api/resources.py

```diff
@@ -469,17 +469,18 @@ def __init__(self, *args, entity_name='policy', entity_collection='policies', **
     def url(self) -> str:
         return f"{self.parent.url}/{self._entity_collection}"
 
-    def append(self, policy):
+    def append(self, *policies):
         params = self.list().entity
-        params["policies_config"].append(policy)
+        params["policies_config"].extend(policies)
         params["service_id"] = self.parent["service_id"]
-        self.update(params=params)
+        return self.update(params=params)
 
-    def insert(self, index, policy):
+    def insert(self, index: int, *policies):
         params = self.list().entity
-        params["policies_config"].insert(index, policy)
+        for (i, policy) in enumerate(policies):
+            params["policies_config"].insert(index + i, policy)
```

Comment:
yes, this is the behavior I was thinking about.
I was thinking about current[:index] + policies + current[index:] what results in same outcome